### PR TITLE
Migrate fraud detection

### PR DESCRIPTION
**Pull request by xuex2017**
migrate fraudDetection notebook to analytics-zoo/apps, deleted folder 'notebook'

### CODE REVIEW

**Comment by xuex2017** [comment]
@shane-huang I removed the 'notebook' folder and put the fraudDetection notebook under apps

**Comment by xuex2017** [comment]
jenkins passed: http://172.168.2.101:8080/job/ZOO-PR-Validation/335/console

File: apps/fraudDetection/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+ZOO has been used in the Fraud Detection system for one of the top payment companies. As a highly simplified
+demo, this notebook uses the public data set to build a fraud detection example on Apache Spark.
+
+The notebook is developed using Scala with Apache Spark 2.1 and ZOO 0.1.0. Refer to
```
Comment (jason-dai) [inline apps/fraudDetection/README.md]: Change "Apache Spark 2.1 and ZOO 0.1.0" to "Analytics Zoo for Spark 2.1 and BigDL"

Comment (jason-dai) [reply apps/fraudDetection/README.md]: Remove "Refer to ..."

File: apps/fraudDetection/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+ZOO has been used in the Fraud Detection system for one of the top payment companies. As a highly simplified
```
Comment (jason-dai) [inline apps/fraudDetection/README.md]: Remove the first sentence.

File: apps/fraudDetection/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+1. Refer to [Apache Toree](https://github.com/apache/incubator-toree/blob/master/README.md) for
+how to use scala in Jupyter notebook.
+
+An outline is:
+```bash
+pip install https://dist.apache.org/repos/dist/dev/incubator/toree/0.2.0/snapshots/dev1/toree-pip/toree-0.2.0.dev1.tar.gz
+```
+
+2. Build zoo jar file under Spark 2.x
```
Comment (jason-dai) [inline apps/fraudDetection/README.md]: Change to "Analytics Zoo"

**Comment by xuex2017** [comment]
@hhbyyh please help review this pr, thanks.

File: apps/fraudDetection/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+An outline is:
+```bash
+pip install https://dist.apache.org/repos/dist/dev/incubator/toree/0.2.0/snapshots/dev1/toree-pip/toree-0.2.0.dev1.tar.gz
+```
+
+2. Build Analytics Zoo jar file under Spark 2.x.
+
+3. To support the training for imbalanced data set in fraud detection, some Transformers and algorithms are developed in 
+https://github.com/intel-analytics/analytics-zoo/tree/legacy/pipeline/fraudDetection.
```
Comment (hhbyyh) [inline apps/fraudDetection/README.md]: A pre-built jar file is provided in the same directory with this Readme.

File: apps/fraudDetection/fraudDetection.ipynb [hunk 5]
```diff
@@ -0,0 +1,545 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "By using bagging model, we can increase the AUPRC from 0.71 to 0.93.\n",
+    "\n",
+    "The code used in this notebook can be found from https://github.com/intel-analytics/analytics-zoo/tree/master/pipeline/fraudDetection.\n",
```
Comment (hhbyyh) [inline apps/fraudDetection/fraudDetection.ipynb]: is this link right?

File: apps/fraudDetection/README.md (hunk 4 above)
Comment (xuex2017) [reply apps/fraudDetection/README.md]: fraud-1.0.1-SNAPSHOT.jar is pre-built, but an analytics-zoo jar file needs to be built

File: apps/fraudDetection/fraudDetection.ipynb (hunk 5 above)
Comment (xuex2017) [reply apps/fraudDetection/fraudDetection.ipynb]: I corrected the link. Thx

**Pull request merged**